src/pages/my_page/user_profile/ResearchProfilePanel.tsx

```diff
@@ -21,14 +20,12 @@ interface ResearchProfilePanelProps {
 
 export const ResearchProfilePanel = ({ person, isLoadingPerson }: ResearchProfilePanelProps) => {
   const { t } = useTranslation();
-  const location = useLocation();
 
   const fullName = person?.names ? getFullCristinName(person.names) : '';
   const orcidUri = getOrcidUri(person?.identifiers);
   const activeAffiliations = person?.affiliations ? filterActiveAffiliations(person.affiliations) : [];
-
-  const currentPath = location.pathname.replace(/\/$/, '').toLowerCase();
-  const isPreview = currentPath === UrlPathTemplate.MyPageMyPersonalia;
+  const personBackground = getLanguageString(person?.background) ?? '';
+  const personKeywords = person?.keywords ?? [];
 
```

Comment:
Overflødig ettersom komponenten kun brukes to plasser,  og `Forhåndsvisning` skal vises på begge. Mistenker at den hang igjen fra tidligere tanken om å bruke samme content på både Forskerprofil og forhåndsvisning.

Reply:
fint å få ryddet bort!